docs/source/tags/table.md

```diff
@@ -10,9 +10,10 @@ Use the Table tag to display object keys and values in a table.
 
 ### Parameters
 
-| Param | Type |
-| --- | --- |
-| value | <code>string</code> | 
+| Param | Type | Description |
+| --- | --- | --- |
+| value | <code>string</code> | Data field value containing JSON type for Table |
+| [valueType] | <code>string</code> | Value to define the data type in Table |
```

Comment:
is it the right example?

Reply:
we should definitely improve documentation for Table, even a list of possible valueTypes is missed here. but this PR is for Labels/Choices mostly, other tags here just because they were not updated for a while